### PR TITLE
Final form - send asset

### PR DESCRIPTION
**Pull request by flexxnn**

### CODE REVIEW

File: src/routes/DigitalAssets/routes/SendWizard/components/FirstStepForm/firstStepForm.m.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+/* stylelint-disable */
+.first-step {
```
Comment (Dragonesis) [inline src/routes/DigitalAssets/routes/SendWizard/components/FirstStepForm/firstStepForm.m.scss]: Why are these styles empty?

File: src/routes/DigitalAssets/routes/SendWizard/DigitalAssetsSendWizardContainer.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+const mapDispatchToProps = dispatch => ({
+  close: () => dispatch(push('/digital-assets/grid')),
+
+  // eslint-disable-next-line fp/no-rest-parameters
+  sendTransaction: (...args) => dispatch(sendTransaction(...args)),
+
+  // eslint-disable-next-line fp/no-rest-parameters
+  addPendingTransaction: (...args) => dispatch(addPendingTransaction(...args)),
```
Comment (IViol) [inline src/routes/DigitalAssets/routes/SendWizard/DigitalAssetsSendWizardContainer.js]: don't need to use `dispatch` here

File: src/routes/DigitalAssets/routes/SendWizard/components/FirstStepForm/FirstStepForm.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    if (values.recipient === values.owner) {
+      return t`Recipient address is the same as sender address`
+    }
+
+    return ''
+  }
+
+  handleSubmit = async (values: FirstStepValues) => {
+    try {
+      const validationSchema = yup.object().shape({
```
Comment (IViol) [inline src/routes/DigitalAssets/routes/SendWizard/components/FirstStepForm/FirstStepForm.js]: better to define such schemas in global scope

File: src/routes/DigitalAssets/routes/SendWizard/components/FirstStepForm/firstStepForm.m.scss (hunk 1 above)
Comment (flexxnn) [reply src/routes/DigitalAssets/routes/SendWizard/components/FirstStepForm/firstStepForm.m.scss]: Will be removed in final version

**Pull request closed without merging**